Bio/PDB/PDBIO.py

```diff
@@ -378,6 +378,6 @@ def save(self, file, select=_select, write_end=True, preserve_atom_numbering=Fal
             if model_flag and model_residues_written:
                 fp.write("ENDMDL\n")
         if write_end:
-            fp.write("END\n")
+            fp.write("END   \n")
```

Comment:
Is this what the specification says, or just to avoid a warning from the new parser?

Reply:
[It's what the specification says](https://www.wwpdb.org/documentation/file-format-content/format33/sect11.html#END):
```
1 -  6        Record name    "END   "
```

Reply:
Perfect. Thanks for confirming.